tools/helpers.py

```diff
@@ -205,4 +259,30 @@ def build_rnafusion_command(config, outdir: str, logdir: str, ss_path: str, test
     rnafusion_command.append("--outdir")
     rnafusion_command.append(os.path.join(outdir, "rnafusion"))
 
-    return rnafusion_command
+    return {"nf-core/rnafusion": rnafusion_command}
+
+
+def start_pipe_threads(pipe_dict: dict, logger):
```

Comment:
Functions without an explicit `return` could be type hinted as returning `None`

eg. `def start_pipe_threads(pipe_dict: dict, logger) -> None:` ...

Reply:
Fixed in be54ef5